fix(docs): Update access-credentials.md (#3506)
runatlantis/atlantis: runatlantis.io/docs/access-credentials.md

Only a single installation per GitHub App is supported at the moment.
:::

::: tip NOTE

> GitHub App handles the webhook calls by itself, hence there is no need to create wehbooks separately. If you have created webhooks manually, you need to remove them when using GitHub App. Otherwise, there would be 2 calls to Atlantis which causes locking errors on path/workspace.

Reviewer: ```suggestion
GitHub App handles the webhook calls by itself, hence there is no need to create wehbooks separately. If webhooks were created manually, those should be removed when using GitHub App. Otherwise, there would be 2 calls to Atlantis resulting in locking errors on path/workspace.
```
